session/session.go

```diff
@@ -357,6 +360,15 @@ func (s *session) StoreQueryFeedback(feedback interface{}) {
 	}
 }
 
+// StoreIndexUsage stores index usage information in idxUsageCollector.
+func (s *session) StoreIndexUsage(dbName string, tblName string, idxName string, rowsSelected int64) {
+	if s.idxUsageCollector == nil {
+		return
+	}
+	id := fmt.Sprintf("%s.%s.%s", dbName, tblName, idxName)
+	s.idxUsageCollector.Update(id, &handle.IndexUsageInformation{QueryCount: 1, RowsSelected: rowsSelected})
```

Comment:
How about changing the `Update(id, value)` to `Update(db, tbl, idx, value)`?
Then we can encode them inside it.

Reply:
OK. You are right.

Reply:
It's better to encode them in `indexUsageMap.update` like `indexUsageMap.query`.

Reply:
Got it.